test/anyref.wast.from-wast

```diff
@@ -1,11 +1,11 @@
 (module
- (type $FUNCSIG$aa (func (param anyref) (result anyref)))
+ (type "anyref -> anyref" (func (param anyref) (result anyref)))
```

Comment:
it is slightly unfortunate to print out a type that is not visibly used anywhere, like in this test (i like the change to not print the type on the function, though!)

can be as a followup though if it's not trivial.

Reply:
I actually don't mind having this section because it's easy to ignore if I want but sometimes it's nice to have an overview of all the function types in one place. But changing this would require a separate ParallelAnalysisPass, so lets defer that discussion to a followup.